ee/stargate/internal/server/grpc/api.go

```diff
@@ -147,7 +148,9 @@ func (s *Server) Stargate(stream api.StargateService_StargateServer) error {
 				}
 				wr := entry.(waitingResponse)
 
-				s.metricsRegistry.GRPCLatencies().Record(ctx, time.Since(wr.sendAt).Milliseconds())
+				s.metricsRegistry.GRPCLatencies().Record(ctx, time.Since(wr.sendAt).Milliseconds(), metric.WithAttributes([]attribute.KeyValue{
+					attribute.String("name", fmt.Sprintf("%s.%s", organizationID, stackID)),
```

Comment:
Where is this formatting used for ?

Reply:
Will only be used in grafana for @flemzord to have more precise metrics and visual about what happened for stacks